src/backends/fvm.hpp

```diff
@@ -4,4 +4,14 @@
 
 #ifdef NMC_HAVE_CUDA
     #include <backends/gpu/fvm.hpp>
+#else
+// FIXME: intermediate fix as part of virtualization of cell_group.
+// This requires that a runtime choice can be made during model set up
+// about which backend is to be used group by group. We use the multicore
+// backend as a dummy GPU back end when compiled without gpu support so that
+// this decision does not require preprocessor directives in model.hpp.
+namespace nest { namespace mc { namespace gpu {
+    using backend = nest::mc::multicore::backend;
+}}} // namespace nest::nc::gpu
```

Comment:
a `null_backend` has been added
  * derives from multicore back end
  * always returns `false` when queried if it implements a specific mechanism
  * throws if the user attempts to make a mechanism instance
  * new `static bool is_supported()` method returns `false`
Also added a simple unit test to check that the `null_backend` is used when not compiled with gpu support.